_meeting-notes/2019-12-10.md

```diff
@@ -0,0 +1,150 @@
+Testground Weekly (45 mins)
+===
+
+###### tags: `IPFS` `Testing`
+
+:::info
+- **Host:** @daviddias
+- **Location:** https://protocol.zoom.us/j/181391682
+- **Date:** Dec 10, 2019 6:00 PM (UTC)
+- **Participants:**
+    - @raulk
+    - @daviddias
+    - @nonsense
+    - @hacdias
+    - @jimpick
+    - @stebalien
+- **Agenda**
+  ~~1. Sprint Update `00min`~~
+  2. Identify blockers / Answer Questions `30min`
+  3. This week's planning `15min`
+- **Reference:** 
+  - [Last week meeting minute 2019-12-03](https://github.com/ipfs/testground/blob/master/_meeting-notes/2019-12-03.md)
+  - [HOW_WE_WORK.md#weekly-sync](https://github.com/ipfs/testground/blob/master/docs/HOW_WE_WORK.md#weekly-sync)
+  - [KanBan Board](https://app.zenhub.com/workspaces/test-ground-5db6a5bf7ca61c00014e2961/board)
+  - [Testing Infra Design](https://github.com/ipfs/testground/blob/master/docs/SPEC.md)
+  - [Testing Inventory](https://github.com/ipfs/testground/blob/master/docs/test-inventory.md)
+  - [Testing Improvement Tracker](https://docs.google.com/spreadsheets/d/1xyqyGUF-oe3x9ln88YonVeOMWWdknik74lVgL_3dBY8/edit#gid=0)
+
+:::
+
+## :mega: Update (Done, Shipped, Made Progress)
+> To be reviewed Async
+
+- ==@raulk==
+- ==@daviddias==
```

Comment:
@daviddias (me) from @raulk from [last week's update](https://github.com/ipfs/testground/blob/master/_meeting-notes/2019-12-03.md#dart-next), what is missing is:

- [ ] ~~Collect architecture decisions that need to be captured and written down~~ 
  - We are now in the progress of benchmarking Docker Swarm vs. Kubernetes (owner: @nonsense)
  - I missed to inquiry everyone and compile this list. The preliminary list is:
    - We need to document how Sidecar is integrated https://github.com/ipfs/testground/issues/221
    - We need describe where is the line that separates Testground from a Test Plan https://github.com/ipfs/testground/issues/224